tests/functional/analyze/test_analyze.py

```diff
@@ -91,6 +91,71 @@ def __analyze_incremental(self, content_, build_json, reports_dir,
                 os.remove(f)
         self.assertEquals(failed_file_count, failed_count)
 
+    def test_compler_info_files(self):
+        # GIVEN
+        build_json = os.path.join(self.test_workspace, "build_simple.json")
+        reports_dir = os.path.join(self.test_workspace, "reports")
+        source_file = os.path.join(self.test_workspace, "simple.cpp")
+
+        # Create a compilation database.
+        build_log = [{"directory": self.test_workspace,
+                      "command": "gcc -c " + source_file,
```

Comment:
Due to `gcc` not existing on macOS systems, this way of creating a build log on the fly is prone to break macOS tests. See #824.

Reply:
I thought gcc is a symbolic link to clang on macOS. https://stackoverflow.com/questions/19535422/os-x-10-9-gcc-links-to-clang
Has this changed in systems newer than El Capitan?

Reply:
I've no macOS machine, so I'm not sure. There was definitely no such thing as `gcc` and `g++` on the Travis macOS instances a few months ago.

Reply:
I am not sure if this is really a problem since there are several other uses like this in other test cases:
```
tests/functional/analyze/test_analyze.py|102 col 35| "command": "gcc -c " + source_file,
tests/functional/analyze/test_analyze.py|172 col 35| "command": "gcc -c " + source_file,
tests/functional/analyze/test_analyze.py|205 col 35| "command": "gcc -c " + source_file,
tests/functional/analyze/test_analyze.py|240 col 35| "gcc -c " + source_file)    
tests/functional/analyze/test_analyze.py|303 col 35| "command": "gcc -c " + source_file,
tests/functional/ctu/test_files/buildlog.json|4 col 17| "command": "gcc -c lib.c -o lib.o",  
tests/functional/ctu/test_files/buildlog.json|9 col 17| "command": "gcc -c main.c -o main.o",
```

Reply:
in this case compilation.json is not created on the fly, but wired in into the test case. So it will not cause problem on Mac.